spec/factories.rb

```diff
@@ -248,4 +248,12 @@
     content { "(#{Faker::Lorem.sentence})[/#{File.join(Faker::Lorem.words)}]" }
     coronavirus_page
   end
+
+  factory :live_stream do
+    url { "https://www.youtube.com/watch?v=NiplUCnwc5A" }
+    formatted_stream_date { "1 April 2020" }
+    trait :without_validations do
```

Comment:
`without_validations`:

There is url validation on the livestream model, which meant adding a stub every time we create a payload for publishing api in a spec. This isn't really representative of how the app works, because there is a livestream object in the database, so we are just fetching it rather than creating it.